sky/packages/sky/example/game/lib/game_demo_world.dart

```diff
@@ -320,6 +321,11 @@ class GameDemoWorld extends NodeWithSize {
     if (_numFrames > _numFramesShieldActive) _shield.visible = false;
     else if (_numFrames > _numFramesShieldActive - _numFramesShieldFlickers) _shield.visible = !_shield.visible;
 
+    // Check for exit back to main screen
+    if (_isGameOver && _numFrames - _gameOverFrame == 60) {
```

Comment:
I'm pretty much counting on the game running 60 fps. A completely new game is created when restarting the game, so no need for any resets.